rocketchat-core/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 group 'com.rocketchat.core'
-version '0.7.2'
+version '0.7.1.9'
```

Comment:
You are downgrading the version, it should be `0.7.2.x` or `0.7.3` or `0.8.0`

Reply:
Will add it